# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/PVR.ipdl
# Commit: 9c0c9e80f309
# Full Hash: 9c0c9e80f30955b6fdb3a2995ac7d7299ef30342
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 2
# Description:
#   Bug 1550422 - P9. Sync preferences in VR process when they change. r=daoshengmu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31017
# ==============================================================================

diff -r 10c153ddbaea -r 9c0c9e80f309 gfx/vr/ipc/PVR.ipdl
--- a/gfx/vr/ipc/PVR.ipdl	Wed May 22 13:23:07 2019 +0000
+++ b/gfx/vr/ipc/PVR.ipdl	Wed May 22 12:42:33 2019 +0000
@@ -10,6 +10,7 @@
 
 include GraphicsMessages;
 include MemoryReportTypes;
+include PrefsTypes;
 include protocol PVRGPU;
 
 include "VRMessageUtils.h";
@@ -26,6 +27,7 @@
 
   async UpdatePref(GfxPrefSetting pref);
   async UpdateVar(GfxVarUpdate var);
+  async PreferenceUpdate(Pref pref);
   async OpenVRControllerActionPathToVR(nsCString aPath);
   async OpenVRControllerManifestPathToVR(OpenVRControllerType aType, nsCString aPath);
   async RequestMemoryReport(uint32_t generation,